index.html

```diff
@@ -413,55 +413,78 @@ <h3>Proofs</h3>
         </p>
 
         <p>
-A <a>data integrity proof</a> typically includes at least the following
-attributes:
+The following attributes are defined for use in a <a>data integrity proof</a>:
         </p>
 
         <dl style="margin-left: 1em;">
-          <dt><dfn>type</dfn><dt>
+          <dt><dfn>type</dfn></dt>
           <dd>
-<em>Required</em>. The specific <a>proof type</a> used. For example, an
-<code>Ed25519Signature2020</code> type indicates that the proof includes a
-digital signature produced by an <code>ed25519</code> cryptographic key.
+The specific <a>proof type</a> used for the cryptographic proof MUST be
+specified as a string that maps to a URL [[URL]]. Examples of proof types
+include <code>DataIntegrityProof</code> and <code>Ed25519Signature2020</code>.
```

Comment:
I wonder whether the URLs to which these example types map, or how that mapping is to be achieved in general, shouldn't also be included?

Reply:
Yes, it should be included. I have raised issue https://github.com/w3c/vc-data-integrity/issues/60 to make sure we have a discussion about this before I suggest some text. I believe the solution is straight forward, but we all know how that goes... better to get some preliminary buy-in via an issue before charging forth with some language that might be controversial.